service/src/test/java/uk/gov/hmcts/reform/fpl/model/JudgeTest.java

```diff
@@ -43,6 +44,35 @@ void shouldReturnAllocatedJudgeLastNameWhenMagistratesIsNotSelected() {
         assertThat(lastName).isEqualTo("Judge Last Name");
     }
 
+    @Test
+    void shouldReturnTrueWhenAllocatedJudgeAndParameterJudgeHaveEqualFields() {
```

Comment:
What do you think of` shouldReturnTrueWhenJudgeAndJudgeAndLegalAdvisorHaveEqualFields`?

Reply:
shouldReturnTrueWhenJudgesHaveEqualFields? 👍